addons/website_forum/models/forum.py

```diff
@@ -126,6 +126,8 @@ def _get_default_faq(self):
     karma_editor = fields.Integer(string='Editor Features: image and links',
                                   default=30, oldname='karma_editor_link_files')
     karma_user_bio = fields.Integer(string='Display detailed user biography', default=750)
+    karma_post = fields.Integer(string='Ask questions without validation', default=100)
+    karma_moderate = fields.Integer(string='Moderate posts', default=1000)
```

Comment:
Small remark: we have a "karma_flag" at 500; maybe being able to moderate at 100 is a bit low :) .